config/software/git.rb

```diff
@@ -51,6 +50,7 @@
     "NEEDS_LIBICONV"       => "1",
     "NO_GETTEXT"           => "1",
     "NO_PYTHON"            => "1",
+    "NO_PERL"              => "1",
```

Comment:
Let's add a nice comment above this line and also point to https://github.com/git/git/blob/master/INSTALL#L102-L109